modules/collect/src/test/java/com/opengamma/strata/collect/TestHelper.java

```diff
@@ -521,14 +518,14 @@ public void flush() {
    */
   public static void assertUtilityClass(Class<?> clazz) {
     assertNotNull(clazz, "assertUtilityClass() called with null class");
-    assertTrue(Modifier.isFinal(clazz.getModifiers()), "Utility class must be final");
+    assertEquals(Modifier.isFinal(clazz.getModifiers()), true, "Utility class must be final");
```

Comment:
Is it worth defining `assertTrue`? It's such a standard method. `assertEquals(..., true)` seems a bit odd.

Reply:
These are private methods, and since they are only used in a couple of places, it didn't seem worthwhile.